staging/src/k8s.io/client-go/tools/clientcmd/client_config.go

```diff
@@ -604,14 +604,14 @@ func (config *inClusterClientConfig) Possible() bool {
 		err == nil && !fi.IsDir()
 }
 
-// BuildConfigFromFlags is a helper function that builds configs from a master
+// BuildConfigFromFlags is a helper function that builds configs from a control plane
 // url or a kubeconfig filepath. These are passed in as command line flags for cluster
-// components. Warnings should reflect this usage. If neither masterUrl or kubeconfigPath
+// components. Warnings should reflect this usage. If neither controlPlaneUrl or kubeconfigPath
 // are passed in we fallback to inClusterConfig. If inClusterConfig fails, we fallback
 // to the default config.
-func BuildConfigFromFlags(masterUrl, kubeconfigPath string) (*restclient.Config, error) {
-	if kubeconfigPath == "" && masterUrl == "" {
-		klog.Warningf("Neither --kubeconfig nor --master was specified.  Using the inClusterConfig.  This might not work.")
+func BuildConfigFromFlags(controlPlaneUrl, kubeconfigPath string) (*restclient.Config, error) {
+	if kubeconfigPath == "" && controlPlaneUrl == "" {
+		klog.Warningf("Neither --kubeconfig nor --controlplane was specified.  Using the inClusterConfig.  This might not work.")
```

Comment:
I don't actually think there's a `master` flag registered by client go... this function doesn't know the flags used by the caller to register the server URL or kubeconfig args. Suggest this instead (mapping more closely to the typical `--kubeconfig` and `--server` CLI flags client-go registers:

```suggestion
		klog.Warningf("No kubeconfig file or server URL was specified.  Using the inClusterConfig.  This might not work.")
```